website_cookie_notice/website.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<openerp>
+<data>
+    <template id="cookiebanner" name="cookiebanner" inherit_id="website.theme">
```

Comment:
>
> Better if you inherit from website.assets_frontend, because then Odoo
> joins and reduces al JS and load at the end of body tag (if not ?debug= in
> URL)
>
>
>
Changed, thanks